recipes-connectivity/opendds/opendds_3.24.bb

```diff
@@ -0,0 +1,29 @@
+# Specify SHA-1 for the release to avoid constantly checking the upstream repo.
+
+SRCREV = "4dd8c404253b8db95bfe3e98ea38a1b14ef228a8"
```

Comment:
The revision for the current release of OpenDDS is 9ededb80594119d8d54b1622cc68f4666b1479a5.  You can find this using the latest-release branch or the most recent release tag, which today is DDS-3.24.1.  Subsequent commits on branch-DDS-3.24 are not part of a release.

Reply:
resolved with https://github.com/OpenDDS/meta-opendds/pull/23/commits/e736f5d6917daedf82c534da5a90a7981ee92797